src/components/Login/Login.tsx

```diff
@@ -0,0 +1,144 @@
+import "./Login.css";
+import React, { useReducer, useState } from "react";
+
+// username: "mor_2314",
+// password: "83r5^_",
+const URL = "https://fakestoreapi.com/auth/login";
+const userData = ["User123", "abcd1234"];
+
+interface LoginProps {
```

Comment:
Same here. User separate file for interfaces, enums, types etc.

Reply:
addressed the comment.